### PR TITLE
improve linkage error message

### LINKED ISSUE #409: 3 group(s) of 9 static linkage error(s)
We can improve this message further. In particular something like "9 methods in 3 classes are not found."

image

Also, there's no need for (s) plurals. It's straightforward to code a check for whether there's 0 1, or more than one such things.

### PR DESCRIPTION
**Pull request by elharo**
@suztomo fix #409

### CODE REVIEW

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/JarLinkageReportTest.java [hunk 1]
```diff
@@ -133,10 +134,28 @@ public void testToString() {
   @Test
   public void testFormatByGroup() {
     Assert.assertEquals(
-        "3 group(s) of 4 static linkage error(s)\n"
+        "4 missing members in 3 classes\n"
```
[the remark this answers is not in the document]
Comment (elharo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/JarLinkageReportTest.java]: what do you suggest? I'm not sure of a word that covers classes, field, and methods. Perhaps just errors?

[the remark this answers is not in the document]
Comment (suztomo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/JarLinkageReportTest.java]: I would use "problems". For example, "3 problems causing 4 linkage errors"

[the remark this answers is not in the document]
Comment (elharo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/JarLinkageReportTest.java]: problems and errors are two similar. I think 3 classes is accurate here. Maybe 4 errors in 3 classes,

[the remark this answers is not in the document]
Comment (elharo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/JarLinkageReportTest.java]: I think I have it. This is four missing *names* in three classes. There's no static linkage in Java as @chanseokoh has pointed out. What we're not finding in all three cases are dynamically referenced *names*. I may work on some of the JLBP and glossary to clarify this.

[the remark this answers is not in the document]
Comment (suztomo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/JarLinkageReportTest.java]: The test has only lines for not-found cause (`SYMBOL_NOT_FOUND` and `CLASS_NOT_FOUND`). There are other cases for the grouping (`LinkageErrorCause`). For example:

- "ClassX has changed incompatibly. " (INCOMPATIBLE_CLASS_CHANGE)
- "ClassY.methodZ is not accessible. " (INACCESSIBLE_MEMBER)

They are not "missing names". You may need to switch the words based on `LinkageErrorCause.getReason()`.

**Pull request merged**